src/pip/_vendor/pep517/_in_process.py

```diff
@@ -238,12 +238,22 @@ def build_sdist(sdist_directory, config_settings):
         raise GotUnsupportedOperation(traceback.format_exc())
 
 
+def build_editable():
```

Comment:
IIRC, as per [Pip's Vendoring Policy](https://pip.pypa.io/en/latest/development/vendoring-policy/) , the changes in the vendored libraries should be done in the original vendor repo, and re-vendored here to apply those changes to pip.

Reply:
Thanks. When this progresses beyond a prototype I will follow that policy.